src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -261,33 +261,6 @@ def stop_amqp_receiver(receiver, task):
     task.join(5)
 
 
-def init_rabbitmq():
-    """ start a rabbitmq broker """
-    hostname = get_ip()
-    try:
-        # first try to stop any existing process
-        subprocess.call(['sudo', 'rabbitmqctl', 'stop'])
-        time.sleep(5)
-        proc = subprocess.Popen(['sudo', '--preserve-env=RABBITMQ_CONFIG_FILE', 'rabbitmq-server'])
```

Comment:
for the amqp ssl tests to work, you must be able to start rabbitmq-server with the RABBITMQ_CONFIG_FILE env variable set to the conf file generated in `test_ps_s3_creation_triggers_on_master_ssl()`.

probably the best option is to move the SSL file creation into rabbitmq.py